daml-script/test/src/test-utils/com/daml/lf/engine/script/test/AbstractFuncIT.scala

```diff
@@ -183,8 +193,18 @@ abstract class AbstractFuncIT
           )
         } yield {
           assert(vals.size == 2)
-          assert(vals.get(0) == SParty(Party.assertFromString("carol")))
-          assert(vals.get(1) == SParty(Party.assertFromString("dan")))
+          inside(vals.get(0)) { case SParty(partyId) =>
+            inside(partyId.split("::")) { case Array(prefix, suffix) =>
+              prefix shouldBe "carol"
+              suffix.length shouldBe 68
```

Comment:
not sure this assertion is super useful.

Reply:
We should review the test.